nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -110,6 +174,10 @@
     empty_dirs:
       - "."
 
+- module-name: "curses"
+  implicit-imports:
+    - depends:
+        - "_curses"
```

Comment:
That's actually a stdlib module that we overlooked before.

Reply:
I think on Windows, curses doesn't work until something from pip is installed, that kind of messes things up.